test/support/browser_helper.rb

```diff
@@ -28,6 +35,20 @@ class BrowserTestCase < Minitest::Spec
   end
 
   def setup
+    debug = ENV['DEBUG'] == '1'
```

Comment:
I should expect DEBUG to print phantomjs native debug stuff as well, right?  (Seeing just a ton of "WebPage - evaluateJavaScript ..." logging).

Reply:
Oh wait, i see you're passing this debug into phantom just below.  Nevermind!